src/js/dataMapping/navigation/menuOptions.jsx

```diff
@@ -38,6 +45,11 @@ export const profileOptions = [
         label: 'Recipients',
         url: '#/recipient',
         enabled: true
+    },
+    {
+        label: <New />,
+        url: '#/disaster/covid-19',
+        enabled: true
```

Comment:
This needs to be dark released.


```suggestion
        enabled: kGlobalConstants.CARES_ACT_RELEASED
```